sources/packages/web/src/views/student/StudentAccountActivity.vue

```diff
@@ -11,7 +11,7 @@
       />
     </template>
     <student-account-restrictions />
-    <student-scholastic-standing-limited-history />
+    <student-scholastic-standing-limited-history :showPartTimeSummary="false" />
```

Comment:
Please use the kebab case for Vue component attributes. `:show-part-time-summary="false"`